draft/overlay/oledbonnet.md

```diff
@@ -0,0 +1,80 @@
+﻿
+<!--
+---
+name: OLED Bonnet
+class: board
+type: Display
+formfactor: pHAT
+manufacturer: Adafruit
+description: A 128x64 display with jostick and buttons for your Pi
+url: https://learn.adafruit.com/adafruit-128x64-oled-bonnet-for-raspberry-pi
+github: https://github.com/adafruit/Adafruit_Python_SSD1306
+buy: https://www.adafruit.com/product/3531
+image: 'oledbonnet.png'
+pincount: 40
+eeprom: no
+power:
+  '1': 
+ground:
+  '6':
+  '9':
+  '14':
+  '20':
+  '25':
+  '30':
+  '34':
+  '39':
+
+pin:
+  '3':
+    mode: i2c
+  '5':
+    mode: i2c
+  
+  ’27’: 
+    
+    name: left
+  
+  ’23’: 
+    
+    name: right
+  
+  ‘4’: 
+    name:center
+  
+  ’17’: 

```

Comment:
hum, OK, I'll assume you intended bcm notations everywhere since the joystick center pin would also fall on a power pin in physical notation.